dapp/src/assets/icons/index.ts

```diff
@@ -2,7 +2,7 @@ export * from "./Info"
 export * from "./ArrowUpRight"
 export * from "./AcreLogo"
 export * from "./Pause"
-export * from "./LoadingSpinnerSuccessIcon"
```

Comment:
Why did you change the type of export here? It brings inconsistency against other modules.

Reply:
I changed the type because because we should use default exports. More info you can find [here](https://github.com/thesis/acre/commit/4b825290f930f3c934e879c8a08ebb693f0c7933#discussion_r1624438894).

I believe that soon we will have some time to update it everywhere.